### DISCUSSION TITLE
Truncated text on x-axis for XY chart when using angle prop

### DISCUSSION THREAD

**Discussion by dalyemmett** [airbnb/visx#1414] [Q&A] [answered]
Hey guys, 

Back again.

I've another issue. When I have a really long label for a x-axis tick and I add a rotation angle for the tick, the label text is clipped by the parent boundaries.

I've updated a [codesandbox here](https://codesandbox.io/s/aged-meadow-yl76j?file=/src/App.js)

Reference image: 
![image](https://user-images.githubusercontent.com/97095582/149533260-d3134015-49e4-44bc-a5dd-bd0c06abee26.png)

Any ideas how I can increase the x-axis space?

**Comment by williaster** [comment, marked as answer]
Hey @dalyemmett you just need to adjust the chart `margin` ([docs](https://airbnb.io/visx/docs/xychart#XYChart_margin))

https://codesandbox.io/s/ecstatic-rosalind-kk8ic?file=/src/App.js
